packages/design-system/src/components/VerticalNav/VerticalNav.jsx

```diff
@@ -73,7 +73,7 @@ VerticalNav.propTypes = {
   /**
    * An array of `VerticalNavItem` data objects
    */
-  items: PropTypes.arrayOf(PropTypes.shape(PropTypes.object)).isRequired,
```

Comment:
Could you try and use this: 
`items: PropTypes.arrayOf(PropTypes.shape(VerticalNavItem.propTypes)),`
similar to `VerticalNavItem` proptype

Reply:
That change will work on the site too - however, running`yarn test` with this will result in a VerticalNavItem error:

```
 FAIL  packages/design-system/src/components/VerticalNav/VerticalNavItem.test.jsx
  ● Test suite failed to run

    TypeError: Cannot read property 'propTypes' of undefined

      74 |    * An array of `VerticalNavItem` data objects
      75 |    */
    > 76 |   items: PropTypes.arrayOf(PropTypes.shape(VerticalNavItem.propTypes)),
```

Reply:
ok this works for me then!